go.mod

```diff
@@ -78,7 +78,7 @@ require (
 	github.com/cisco-ie/nx-telemetry-proto v0.0.0-20190531143454-82441e232cf6
 	github.com/containerd/cgroups v1.0.1 // indirect
 	github.com/containerd/containerd v1.5.7 // indirect
-	github.com/coreos/go-semver v0.3.0
+	github.com/coreos/go-semver v0.3.0 // indirect
```

Comment:
go-semver was also already an indirect dependency. Or do I see this wrong?

Reply:
You are correct... Anyway, the used library also provides "greater than" and has a more consistent handling. They do not differ much anyway. :-)

Reply:
Err, if you want `A > B`, you can just do `versionB.LessThan(versionA)` but I'm fine with it.

Reply:
Right, I didn't notice we are using go-semver already. Do you want me to revert to go-semver?

Reply:
That last one seems to be more actively maintained?

Reply:
Reverted.